docs/design/AWS_API_Compatibility.md

```diff
@@ -71,16 +74,32 @@ _* Note that it is also possible for actions to be supported but absent because
-|                       | Users                           | CreateUser                        | ❌            | ✅    | ❌      | ❌        |                                                                         |
-|                       | Users                           | UpdateUser                        | ❌            | ✅    | ❌      | ❌        |                                                                         |
-|                       | Users                           | DeleteUser                        | ❌            | ✅    | ❌      | ❌        |                                                                         |
-|                       | Users                           | ListUsers                         | ❌            | ✅*   | ❌      | ❌        | *No pagination support                                                  |
-|                       | Access Keys                     | GetAccessKeyLastUsed              | ❌            | ✅*   | ❌      | ❌        | *Partially implemented                                                  |
-|                       | Access Keys                     | CreateAccessKey                   | ❌            | ✅    | ❌      | ❌        |                                                                         |
-|                       | Access Keys                     | UpdateAccessKey                   | ❌            | ✅    | ❌      | ❌        |                                                                         |
-|                       | Access Keys                     | DeleteAccessKey                   | ❌            | ✅    | ❌      | ❌        |                                                                         |
-|                       | Access Keys                     | ListAccessKeys                    | ❌            | ✅*   | ❌      | ❌        | *No pagination support                                                  |
 
-_* IAM API uses a different port than the S3 API, and needs to be manually enabled prior to use._
\ No newline at end of file
+
+## IAM and STS API
+The following table compare the actions of managing entities between NooBaa deployments:
+1. NooBaa non-containerized deployment (NC)
+2. Containerized deployment
+
+**IAM API uses a different port than the S3 API (and needs to be manually enabled before use in NC deployments).**
+
+| Category              | Feature                         | API Action                        | Containerized  | NC   | Comments                                                                |
+|:---------------------:|:-------------------------------:|:---------------------------------:|:-------------:|:------:|-------------------------------------------------------------------------|
+| **STS API** | Session Tokens | AssumeRole           | ✅ | ❌ | |
```

Comment:
@jackyalbo could you look at this line?

1. I also moved STS (and not just IAM) to a deployment comparison, if you don't think it makes sense, I would move only IAM.
2. I also thought that you added `AssumeRoleWithWebIdentity` and maybe you want to add this one also.

Reply:
As we are using AssumeRoleWithWebIdentity just for LDAP which is not supported by AWS, I think we can skip it here. 
About 1, I don't really mind putting STS here also - and as you know much more about how much of STS we support, I'll let you take the call :).

Reply:
Thanks, I will leave it as I suggested here.